packages/venia-ui/lib/components/Button/button.css

```diff
@@ -56,7 +56,6 @@
 .root_lowPriority {
     composes: root;
     border-width: 0;
-    color: rgb(var(--venia-error));
```

Comment:
I'm sure there's a reason for removing this, could you explain why? As this'll change all components throughout Venia.

Reply:
Color of the "link" button should be #212121, according to recommendation from @soumya-ashok

Reply:
Just discussed with @soumya-ashok. This change is fine.